tensorrt_llm/llmapi/llm_args.py

```diff
@@ -973,7 +966,7 @@ def model_post_init(self, __context: Any):
         if self.moe_expert_parallel_size is None:
             self.moe_expert_parallel_size = -1
 
-        self.parallel_config = _ParallelConfig(
+        self._parallel_config = _ParallelConfig(
```

Comment:
This might be another candidate for `PrivateAttr`?

Reply:
Yes, I've updated it as a PrivateAttr.